CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## v1.0.0 - 2024-02-01 - Support for DNSv2 API and structural project changes
```

Comment:
Don't want to call something w/changes this large 1.0/stable. Can do 0.99/beta to allow for patches and if things are stable for a bit it can be bumped to 1.0

```suggestion
## v0.99.0 - 2024-02-01 - Support for DNSv2 API and structural project changes
```